Sources/AblyAssetTrackingPublisher/Services/DefaultLocationService.swift

```diff
@@ -4,7 +4,7 @@ import MapboxDirections
 import AblyAssetTrackingCore
 
 class DefaultLocationService: LocationService {
-    private let locationDataSource: PassiveLocationDataSource
+    private let locationDataSource: PassiveLocationManager
```

Comment:
We renamed the type, but shouldn't the variable also be renamed to match?

Reply:
9735a96153647b964a04ea66863d2c9403a799d5